app/services/integrations/slack/incoming_message_builder.rb

```diff
@@ -93,7 +97,18 @@ def private_note?
   def create_message
     return unless conversation
```

Comment:
There seems to be a code climate issue here. could refactor this method into smaller chunks

Reply:
Fixed.